examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.java

```diff
@@ -95,38 +96,34 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onStart() {
         super.onStart();
         user = SyncUser.currentUser();
-        if (user != null) {
-            // Create a RealmConfiguration for our user
-            SyncConfiguration config = new SyncConfiguration.Builder(user, REALM_URL)
-                    .initialData(new Realm.Transaction() {
-                        @Override
-                        public void execute(Realm realm) {
-                            realm.createObject(CRDTCounter.class, 1);
-                        }
-                    })
-                    .build();
-
-            // This will automatically sync all changes in the background for as long as the Realm is open
-            realm = Realm.getInstance(config);
-
-            counter = realm.where(CRDTCounter.class).findFirstAsync();
-            counter.addChangeListener(new RealmChangeListener<CRDTCounter>() {
-                @Override
-                public void onChange(CRDTCounter counter) {
-                    if (counter.isValid()) {
-                        counterView.setText(String.format(Locale.US, "%d", counter.getCount()));
-                    } else {
-                        counterView.setText("-");
+        if (user == null) { return; }
+
+        // Create a RealmConfiguration for our user
+        SyncConfiguration config = new SyncConfiguration.Builder(user, REALM_URL)
+                .initialData(new Realm.Transaction() {
+                    @Override
+                    public void execute(@Nonnull Realm realm) {
+                        realm.createObject(CRDTCounter.class, user.getIdentity());
```

Comment:
I'm surprised this works `createObject(Class, Object)` should only work if a primary key is defined?

Reply:
@cmelchior `CRDTCounter` seems to have the primaryKey (`name`).

Reply:
Ups, missed that above the edits